chart/githubconnector/templates/applicationmapping.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
```

Comment:
We should not create Application Mapping yet - this binds the Application (GitHub Connector, and thus - access to GitHub API via Gateway) to given namespace. This should be done by a client as they decide where they want the application to be accessible from.